libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -443,7 +452,7 @@ WaveTrack::DoGetInterval(size_t iInterval)
          pClip1;
```

Comment:
It might be nicer that way, but for now, the correctness of these transformations does not depend on sorting.

Sorting with each access may be too expensive and keeping sortedness as an invariant would need too much other review to be sure of its completeness.

Reply:
Wherever clips are iterated before, they will be iterated in the same sequence through this new layer of API.  That is more conservative, and one less source of surprising behavior changes.